pyproject.toml

```diff
@@ -0,0 +1,159 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "newa"
+dynamic = ["version"]
+description = "newa"
+readme = "README.md"
+# TODO: provide correct values
+# license = "MIT"
+# license-files = { paths = ["LICENSE"] }
+# requires-python = ">=3.9"
+# authors = [
+#     { name = "...", email = "..." },
+#     ]
+# maintainers = [
+#     { name = "...", email = "..." },
+#     ]
```

Comment:
@kkaarreell who would be authors/maintainers?

Reply:
@happz I think we can populate the initial set with names of current contributors (you, Miro, Patrik, me, possibly Lukas) and extend it later.